packages/analytics-node/src/node-client.ts

```diff
@@ -0,0 +1,142 @@
+import { AmplitudeCore, Destination } from '@amplitude/analytics-core';
```

Comment:
python has [flush](https://developers.amplitude.com/docs/python-beta#flush) and [shutdown](https://developers.amplitude.com/docs/python-beta#shutdown)

Reply:
Added flush, but shutdown might not work for node sdk

Reply:
@bohan-amplitude any thoughts here?